pkg/ipcache/cidr.go

```diff
@@ -34,6 +34,8 @@ import (
 //
 // Upon success, the caller must also arrange for the resulting identities to
 // be released via a subsequent call to ReleaseCIDRIdentitiesByCIDR().
+//
+// DEPRECATED. Prefer UpsertLabels() or OverrideIdentity() instead.
```

Comment:
How come we are recommending `OverrideIdentity()`? I understand that it's part of the new API, but I think we should be carefully review any usages of `OverrideIdentity` because in a way, it's similar to the old API's semantics and delegation to each caller.

Reply:
Good point yeah, I'll remove this and add a note to `OverrideIdentity` to explain that using that API could cause feature conflicts with any users of UpsertLabels, UpsertMetadata, etc., so should be avoided wherever possible if any other subsystems may be using the same IPs.